### PR TITLE
Update Models API for Keras CV

### PR DESCRIPTION
**Pull request by sachinprasadhs**
* Update Models API for all the models, backbones, tasks.

* Arrange the APIs in alphabetical order.

* Created PR in Keras CV for the symbols to be exported https://github.com/keras-team/keras-cv/pull/2389 this needs to be reviewed and merged for keras.io to render these APIs completely.

### CODE REVIEW

File: scripts/cv_api_master.py [hunk 1]
```diff
@@ -306,6 +306,108 @@
...
+            "generate": [
+                "keras_cv.models.DenseNetBackbone",
+                "keras_cv.models.DenseNetBackbone.from_preset",
+                "keras_cv.models.DenseNet121Backbone",
+                "keras_cv.models.DenseNet169Backbone",
+                "keras_cv.models.DenseNet201Backbone",
+            ],
+        },
+        {
+            "path": "efficientnetv1",
```
Comment (fchollet) [inline scripts/cv_api_master.py:334]: Use underscores, e.g. `efficientnet_v1`

File: scripts/cv_api_master.py [hunk 2]
```diff
@@ -306,6 +306,108 @@
...
+                "keras_cv.models.EfficientNetV1B2Backbone",
+                "keras_cv.models.EfficientNetV1B3Backbone",
+                "keras_cv.models.EfficientNetV1B4Backbone",
+                "keras_cv.models.EfficientNetV1B5Backbone",
+                "keras_cv.models.EfficientNetV1B6Backbone",
+                "keras_cv.models.EfficientNetV1B7Backbone",
+            ],
+        },
+        {
+            "path": "efficientnetv2",
```
Comment (fchollet) [inline scripts/cv_api_master.py:350]: Same here

File: scripts/cv_api_master.py [hunk 3]
```diff
@@ -306,6 +306,108 @@
...
+                "keras_cv.models.MiTB0Backbone",
+                "keras_cv.models.MiTB1Backbone",
+                "keras_cv.models.MiTB2Backbone",
+                "keras_cv.models.MiTB3Backbone",
+                "keras_cv.models.MiTB4Backbone",
+                "keras_cv.models.MiTB5Backbone",
+            ],
+        },
+        {
+            "path": "mobilenetv3",
```
Comment (fchollet) [inline scripts/cv_api_master.py:392]: Same here

File: scripts/cv_api_master.py [hunk 4]
```diff
@@ -306,6 +306,108 @@
...
+            "title": "MobileNetV3 backbones",
+            "generate": [
+                "keras_cv.models.MobileNetV3Backbone",
+                "keras_cv.models.MobileNetV3Backbone.from_preset",
+                "keras_cv.models.MobileNetV3SmallBackbone",
+                "keras_cv.models.MobileNetV3LargeBackbone",
+            ],
+        },
+        {
+            "path": "mobilenetv3",
```
Comment (fchollet) [inline scripts/cv_api_master.py:402]: Remove redundant section

File: scripts/cv_api_master.py [hunk 5]
```diff
@@ -387,18 +470,63 @@
...
             "generate": [
-                "keras_cv.models.StableDiffusion",
+                "keras_cv.models.BASNet",
+                "keras_cv.models.BASNet.from_preset",
             ],
         },
         {
-            "path": "retinanet",
-            "title": "The RetinaNet model",
+            "path": "deeplabv3_segmentation",
```
Comment (fchollet) [inline scripts/cv_api_master.py:481]: Same here

**Review by fchollet — COMMENTED** [review]
Thank you for the PR!

**Comment by sachinprasadhs** [comment]
Thank you, made the suggested changes, this PR in CV needs to be merged before merging this https://github.com/keras-team/keras-cv/pull/2389

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Comment by fchollet** [comment]
I'm seeing the following error on CI:

> 6.609 AttributeError: module 'keras_cv.models' has no attribute 'SegFormerB0'

What is the issue?

**Review by divyashreepathihalli — APPROVED** [review]

**Comment by sachinprasadhs** [comment]
> I'm seeing the following error on CI:
> 
> > 6.609 AttributeError: module 'keras_cv.models' has no attribute 'SegFormerB0'
> 
> What is the issue?

I exported few APIs in Keras CV, which needs to be merged here https://github.com/keras-team/keras-cv/pull/2389 for those to be available to process in keras.io

**Comment by sachinprasadhs** [comment]
This can be merged now since the linked Keras -CV PR is merged.

**Pull request merged**